custom_components/tuya_local/devices/hama_radiator_controller.yaml

```diff
@@ -0,0 +1,32 @@
+name: Hama Radiator Controller
+products:
+  - id: 176592/176593
+    name: Hama Radiator Controller
+primary_entity:
+  entity: climate
+  dps:
+    - id: 101
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: true
+          value: "heat"
+          icon: "mdi:radiator"
+        - dps_val: false
+          value: "off"
+          icon: "mdi:radiator-disabled"
+    - id: 102
+      name: current_temperature
+      type: integer
+      mapping:
+        - scale: 10
+          step: 1
```

Comment:
Step is not needed for read-only values, whatever comes from the device will always be used (and anyway 1 is the default for step)